test/e2e/suite/approval/approval.go

```diff
@@ -111,30 +111,48 @@ var _ = framework.CertManagerDescribe("Approval CertificateRequests", func() {
 		}, metav1.CreateOptions{})
 		Expect(err).NotTo(HaveOccurred())
 
+		// Manually create a Secret to be populated with Service Account token
```

Comment:
The changes in this commit are all because `Secret`s for `ServiceAccount`s are no longer autogenerated.